.travis.yml

```diff
@@ -52,7 +53,9 @@ install:
         - travis_retry wget -q https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-7.2.zip
         - unzip -qq sonarqube-7.2.zip
         - travis_retry wget -q https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-7.3.zip
-        - unzip -qq sonarqube-7.3.zip     
+        - unzip -qq sonarqube-7.3.zip
+        - travis_retry wget -q https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-7.4.zip
+        - unzip -qq sonarqube-7.4.zip
```

Comment:
just for your information: sonar-scanner 3.1.0.1141 is not the latest version; The most recent one is https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-3.2.0.1227-linux.zip

Reply:
@ivangalkin thx. Was by intention to do this updates step by step. Makes it easier in case of problems to identify the root cause.